addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -747,6 +762,16 @@ registry.mediaVideo = publicWidget.Widget.extend(MobileYoutubeAutoplayMixin, {
             this._triggerAutoplay(iframeEl);
         });
     },
+    /**
+     * @override
+     */
+    destroy() {
+        if (this.sizeContainerElHidden) {
+            document.removeEventListener("optionalCookiesAccepted", this._showSizeContainerEl);
+            this._showSizeContainerEl;
```

Comment:
You don't need this `sizeContainerElHidden` variable, let's not declare it. You can just call `document.removeEventListener` without checking the event listener was already removed or not.